kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx

```diff
@@ -74,24 +74,26 @@ const Overview: React.FC<Props> = ({
               <td>{leader}</td>
               <td>{offsetMin}</td>
               <td>{offsetMax}</td>
-              <td className="has-text-right">
-                <Dropdown
-                  label={
-                    <span className="icon">
-                      <i className="fas fa-cog" />
-                    </span>
-                  }
-                  right
-                >
-                  <DropdownItem
-                    onClick={() =>
-                      clearTopicMessages(clusterName, topicName, [partition])
+              {!internal ? (
+                <td className="has-text-right">
```

Comment:
I would suggest to hide children of this td instead. In other case to make html of the table valid you need to hide `th` too

Reply:
done